### ISSUE TITLE
Cannot pass an argument `-A` to scripts

### ISSUE THREAD

**Issue by Fenzland** [denoland/deno#3138]
`test.js`
```js
#!/usr/bin/env deno

console.log(Deno.args);
```

```sh
> ./test.js -B
[ "./test.js", "-B" ]
```
```sh
> ./test.js -A
[ "./test.js" ]
```

**Comment by bartlomieju** [comment]
That's because `-A` is an alias for `--allow-all` permission flag.

**Comment by Fenzland** [comment]
> That's because `-A` is an alias for `--allow-all` permission flag.

But I just pass it to the script rather then deno. 

It's unfriendly to let coder to avoid all deno options when design scripts, and change API when deno add a new option that the script happen to use.

**Comment by Fenzland** [comment]
> Yes, agreed. There's already issue for that #3011 (title is different but it's the same underlying problem)

`./foo.js -B -- -A` is also monstrous, maybe `--` also use in script.

**Comment by nayeemrmn** [comment]
Duplicate of #2798.

We seem to have agreed there that simply passing everything after the script name to the script would be most desirable... though nothing's happened.

#3011 is unrelated: it says that `deno` penetrates the `--` when seeking more unnamed arguments. That's not occurring here. The `--` for this usage works fine but people don't like it.

**Comment by bartlomieju** [comment]
This should be fixed after #3389 landed
